demo/summer-2016.html

```diff
@@ -242,7 +242,22 @@
         enableLiveAutocompletion: true
     });
 
-
+  var HashHandler = require("ace/keyboard/hash_handler").HashHandler;
+  keyboardHandler = new HashHandler();
+  keyboardHandler.addCommand({
+  name: 'item.',
+  bindKey: {win: 'Enter', mac: 'Enter'},
+  exec: function(editor) {
+        if(editor.getSession().getContext(editor.getCursorPosition().row) == "list") {
+          editor.insert("\n\t\\item ")
+          return true; // CHANGE HERE ! true is when it capture it.
+        } else {
+          // just leave the key.
+          return false; // CHANGE HERE ! false is when I don't capture it.
+        }},
+  readOnly: true
+});
+editor.keyBinding.addKeyboardHandler(keyboardHandler);
```

Comment:
It is not a good idea, because using editor.commands.addCommand we defining default action with  Enter
In the results, we will lose the opportunity to move to a new line using Enter